starter-core/src/main/java/io/micronaut/starter/build/gradle/GradleDependency.java

```diff
@@ -119,6 +122,14 @@ public String toSnippet() {
         return snippet;
     }
 
+    @NonNull
+    Optional<String> versionCatalogue() {
```

Comment:
I know you like the British version, but I think we should stick with the official terminology which is "version catalog" :)

Reply:
resolved via https://github.com/micronaut-projects/micronaut-starter/pull/1387/commits/0f184501cebbf3992c826e1abf072570ea5e3a6b